backend/open_webui/models/notes.py

```
@@ -109,21 +244,115 @@ def get_notes(
             notes = query.all()
             return [NoteModel.model_validate(note) for note in notes]
 
+    def search_notes(
+        self, user_id: str, filter: dict = {}, skip: int = 0, limit: int = 30
```

Ruff (0.14.8)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)